bluesky/utils/__init__.py

```diff
@@ -592,11 +592,14 @@ def snake_cyclers(cyclers, snake_booleans):
     """
     Combine cyclers with a 'snaking' back-and-forth order.
 
+    If none of the cyclers are "snaked" this is the same as taking the product
+    (*****) of all the cyclers.
```

Comment:
What does `(*****)` mean here?

Reply:
That is attempting to make a bold `*` in rst.